core/icons/comment_icon.ts

```diff
@@ -264,15 +264,16 @@ export class CommentIcon extends Icon implements IHasBubble, ISerializable {
   }
 
   async setBubbleVisible(visible: boolean): Promise<void> {
-    if (visible && (this.textBubble || this.textInputBubble)) return;
-    if (!visible && !(this.textBubble || this.textInputBubble)) return;
+    if (this.bubbleVisiblity === visible) return;
 
     this.bubbleVisiblity = visible;
 
-    if (!this.sourceBlock.rendered || this.sourceBlock.isInFlyout) return;
+    if (!this.sourceBlock.rendered || this.sourceBlock.isInFlyout || this.sourceBlock.isInsertionMarker()) return;
 
     await renderManagement.finishQueuedRenders();
 
+    if (!this.sourceBlock.rendered || this.sourceBlock.isInFlyout || this.sourceBlock.isInsertionMarker()) return;
+
```

Comment:
Ah I missed that these checks were duplicated. It should be fine if you move the `await` line to be the first line in this function! Then you can remove the second check.

Reply:
fixed!